jax/_src/numpy/fft.py

```diff
@@ -23,16 +23,29 @@
 
 
 def _fft_core(func_name, fft_type, a, s, axes, norm):
-  # TODO(skye): implement padding/cropping based on 's'.
   full_name = "jax.numpy.fft." + func_name
-  if s is not None:
-    raise NotImplementedError("%s only supports s=None, got %s" % (full_name, s))
+
+  if s is not None and not isinstance(s, list):
```

Comment:
It looks like `s` is required to be a static sequence of integers here. We could improve validation by using something similar to [`_ensure_index_tuple`](https://github.com/google/jax/blob/5eb07c1b4acd099ff51c29b129950139dfdaf1d9/jax/api_util.py#L41-L46)